internal/services/applications/migrations/application_resource.go

```diff
@@ -331,6 +331,7 @@ func ResourceApplicationInstanceResourceV0() *schema.Resource {
 													"netbios_domain_and_sam_account_name",
 													"sam_account_name",
 													"use_guid",
+													"cloud_displayname",
```

Comment:
This file contains a representation of the schema from an earlier point in time and should not be updated
```suggestion
```